include/globals.h

```diff
@@ -903,14 +985,70 @@ extern RemoteDebug Debug;           // Let everyone in the project know about it
 
     // The mic in the M5 is not quite the same as the Mesmerizer, so it gets a different minimum VU than default
 
-    #define MIN_VU                      280
-    #define NOISE_CUTOFF                1000
-
+     #define MIN_VU                  150
+     #define NOISE_CUTOFF            100
+ 
     #if !(ELECROW)
         #define TOGGLE_BUTTON_1         37
         #define TOGGLE_BUTTON_2         39
     #endif
 
+#elif HELMET
+
+    // This project is set up as a 48x16 matrix of 16x16 WS2812B panels such as: https://amzn.to/3ABs5DK
+    // It uses an M5StickCPlus which has a microphone and LCD built in:  https://amzn.to/3CrvCFh
+    // It displays a spectrum analyzer and music visualizer
+
+    #ifndef PROJECT_NAME
+    #define PROJECT_NAME                    "Helmet"
+    #endif
+
+    #define POWER_LIMIT_MW                  1000
+
+    #define ENABLE_AUDIOSERIAL              0   // Report peaks at 2400baud on serial port for PETRock consumption
+    #define ENABLE_WIFI                     1   // Connect to WiFi
+    #define WAIT_FOR_WIFI                   0   // Hold in setup until we have WiFi - for strips without effects
+    #define TIME_BEFORE_LOCAL               2   // How many seconds before the lamp times out and shows local content
+    #define ENABLE_WEBSERVER                1   // Turn on the internal webserver
+    #define ENABLE_NTP                      1   // Set the clock from the web
+    #define ENABLE_OTA                      0   // Accept over the air flash updates
+    #define ENABLE_REMOTE                   1   // IR Remote Control
+    #define ENABLE_AUDIO                    1   // Listen for audio from the microphone and process it
+    #define COLORDATA_SERVER_ENABLED        0
```

Comment:
I think we can remove this line, considering the `#if USE_PSRAM`/`#else` block that follows which also contains it for both cases.